.github/workflows/test.yml

```diff
@@ -61,7 +61,9 @@ jobs:
             postgresql-${{ matrix.pg }}-pgtap \
             postgresql-${{ matrix.pg }}-postgis-${{ matrix.pgis }} \
             postgresql-${{ matrix.pg }}-postgis-${{ matrix.pgis }}-scripts \
-            postgresql-server-dev-${{ matrix.pg }}
+            postgresql-server-dev-${{ matrix.pg }} \
+            tableversion \
+            dbpatch
```

Comment:
Ok with the merge but note we did NOT need tableversion/dbpatch here (they are installed when needed, under `Test packaging` step). Should not harm, but it's preferrable, I think, to test the code in absence of those packages intially (so we also test that the "master" branches of tableversion/dbpatch repositories do install fine....